requirements.txt

```diff
@@ -21,7 +23,10 @@ MarkupSafe==1.0
 mccabe==0.6.1
 nose==1.3.7
 pep8==1.7.1
+pkg-resources==0.0.0
```

Comment:
Remove this line, It brings about errors when someone trie sto run your aplicarion